layouts/PageWithSideNav/index.tsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react'
+import Head from 'next/head'
+import config from '../../config/config.json'
+
+
+const PageWithSideNav = ({ meta, children }) => (
+  <>
+  <Head>
+    <title>{`${meta.title} - ${config.siteName}`}</title>
```

Comment:
Do we want the guard clause like in GenericLayout?

```
      {meta && meta.title ? 
        (`${meta.title} - ${config.siteName}`)
        : config.siteName
      }
```

Reply:
yep , thanks @tlwr  updated